addons/stock/models/stock_move_line.py

```diff
@@ -762,6 +762,14 @@ def _get_aggregated_properties(self, move_line=False, move=False):
             'packaging': move.product_packaging_id,
         }
 
+    def _compute_packaging_qtys(self, aggregated_move_lines):
```

Comment:
```suggestion
    @api.model
    def _compute_packaging_qtys(self, aggregated_move_lines):
```

Reply:
@Whenrow I've added it
but I still don't know when it should be added or not.
if you can light my candle please :) ?

Reply:
Its only use is for the call from the api that won't check if `self` is empty or not. But in this case as it's a private method I admit there is not real need for this except a kinfd of documentation usage.